scripts/mallet/mallet.gd

```diff
@@ -0,0 +1,10 @@
+extends Node2D
+
+var anim_player: AnimationPlayer
```

Comment:
Here's a nice one-liner!

```suggestion
@onready var anim_player: AnimationPlayer = get_node("AnimationPlayer")
```

Using this would let you delete the `_ready()` function entirely.

Reply:
Yep, just realized this a bit later. Thanks for pointing out!